homeassistant/components/emonitor/config_flow.py

```diff
@@ -0,0 +1,104 @@
+"""Config flow for SiteSage Emonitor integration."""
+import logging
+
+from aioemonitor import Emonitor
+import aiohttp
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.components.dhcp import IP_ADDRESS, MAC_ADDRESS
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.device_registry import format_mac
+
+from . import name_short_mac
+from .const import DOMAIN  # pylint:disable=unused-import
```

Comment:
We no longer need to disable pylint here. It was fixed upstream.

Reply:
done in https://github.com/home-assistant/core/pull/48700